olive/common/constants.py

```diff
@@ -4,9 +4,17 @@
 # --------------------------------------------------------------------------
 from enum import Enum
 
-DEFAULT_WORKFLOW_ID = "default_workflow"
-
 
 class OS(str, Enum):
     WINDOWS = "Windows"
     LINUX = "Linux"
+
+
+############# Engine #############
+
+DEFAULT_WORKFLOW_ID = "default_workflow"
+
+
+############# Packaging #############
+
+BASE_IMAGE = "mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.6-cudnn8-ubuntu20.04"
```

Comment:
The latest ort packages don't support cuda 11.6 anymore. Might have to swich to `"mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.8-cudnn8-ubuntu22.04"` like in https://github.com/microsoft/Olive/pull/1232

Reply:
Updated